packages/keyboard-shortcuts/src/hooks/use-shortcut.native.js

```diff
@@ -0,0 +1,2 @@
+const useShortcut = () => null;
+export default useShortcut;
```

Comment:
Hey @youknowriad, @hypest 
At this moment I don't have a clear context if we should use this function on the gb-mobile or not?
Mobile devices can have external keyboards, but not sure if this function can handle does or it's web-specific?